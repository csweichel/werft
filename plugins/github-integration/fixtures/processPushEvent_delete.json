{
    "Event": {
        "ref": "refs/heads/cw/tbd",
        "before": "37cb8b8f7fc3499bdf65d662793dd2d968ae3bac",
        "after": "0000000000000000000000000000000000000000",
        "repository": {
            "id": 165038235,
            "node_id": "MDEwOlJlcG9zaXRvcnkxNjUwMzgyMzU=",
            "name": "test-repo",
            "full_name": "csweichel/test-repo",
            "private": false,
            "owner": {
                "name": "csweichel",
                "email": "chris@gitpod.io",
                "login": "csweichel",
                "id": 3210701,
                "node_id": "MDQ6VXNlcjMyMTA3MDE=",
                "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
                "gravatar_id": "",
                "url": "https://api.github.com/users/csweichel",
                "html_url": "https://github.com/csweichel",
                "followers_url": "https://api.github.com/users/csweichel/followers",
                "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
                "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
                "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
                "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
                "organizations_url": "https://api.github.com/users/csweichel/orgs",
                "repos_url": "https://api.github.com/users/csweichel/repos",
                "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
                "received_events_url": "https://api.github.com/users/csweichel/received_events",
                "type": "User",
                "site_admin": false
            },
            "html_url": "https://github.com/csweichel/test-repo",
            "description": "This is just a test repo to play with GitHub apps - nothing to see here",
            "fork": false,
            "url": "https://github.com/csweichel/test-repo",
            "forks_url": "https://api.github.com/repos/csweichel/test-repo/forks",
            "keys_url": "https://api.github.com/repos/csweichel/test-repo/keys{/key_id}",
            "collaborators_url": "https://api.github.com/repos/csweichel/test-repo/collaborators{/collaborator}",
            "teams_url": "https://api.github.com/repos/csweichel/test-repo/teams",
            "hooks_url": "https://api.github.com/repos/csweichel/test-repo/hooks",
            "issue_events_url": "https://api.github.com/repos/csweichel/test-repo/issues/events{/number}",
            "events_url": "https://api.github.com/repos/csweichel/test-repo/events",
            "assignees_url": "https://api.github.com/repos/csweichel/test-repo/assignees{/user}",
            "branches_url": "https://api.github.com/repos/csweichel/test-repo/branches{/branch}",
            "tags_url": "https://api.github.com/repos/csweichel/test-repo/tags",
            "blobs_url": "https://api.github.com/repos/csweichel/test-repo/git/blobs{/sha}",
            "git_tags_url": "https://api.github.com/repos/csweichel/test-repo/git/tags{/sha}",
            "git_refs_url": "https://api.github.com/repos/csweichel/test-repo/git/refs{/sha}",
            "trees_url": "https://api.github.com/repos/csweichel/test-repo/git/trees{/sha}",
            "statuses_url": "https://api.github.com/repos/csweichel/test-repo/statuses/{sha}",
            "languages_url": "https://api.github.com/repos/csweichel/test-repo/languages",
            "stargazers_url": "https://api.github.com/repos/csweichel/test-repo/stargazers",
            "contributors_url": "https://api.github.com/repos/csweichel/test-repo/contributors",
            "subscribers_url": "https://api.github.com/repos/csweichel/test-repo/subscribers",
            "subscription_url": "https://api.github.com/repos/csweichel/test-repo/subscription",
            "commits_url": "https://api.github.com/repos/csweichel/test-repo/commits{/sha}",
            "git_commits_url": "https://api.github.com/repos/csweichel/test-repo/git/commits{/sha}",
            "comments_url": "https://api.github.com/repos/csweichel/test-repo/comments{/number}",
            "issue_comment_url": "https://api.github.com/repos/csweichel/test-repo/issues/comments{/number}",
            "contents_url": "https://api.github.com/repos/csweichel/test-repo/contents/{+path}",
            "compare_url": "https://api.github.com/repos/csweichel/test-repo/compare/{base}...{head}",
            "merges_url": "https://api.github.com/repos/csweichel/test-repo/merges",
            "archive_url": "https://api.github.com/repos/csweichel/test-repo/{archive_format}{/ref}",
            "downloads_url": "https://api.github.com/repos/csweichel/test-repo/downloads",
            "issues_url": "https://api.github.com/repos/csweichel/test-repo/issues{/number}",
            "pulls_url": "https://api.github.com/repos/csweichel/test-repo/pulls{/number}",
            "milestones_url": "https://api.github.com/repos/csweichel/test-repo/milestones{/number}",
            "notifications_url": "https://api.github.com/repos/csweichel/test-repo/notifications{?since,all,participating}",
            "labels_url": "https://api.github.com/repos/csweichel/test-repo/labels{/name}",
            "releases_url": "https://api.github.com/repos/csweichel/test-repo/releases{/id}",
            "deployments_url": "https://api.github.com/repos/csweichel/test-repo/deployments",
            "created_at": 1547115470,
            "updated_at": "2022-02-20T14:29:49Z",
            "pushed_at": 1645375122,
            "git_url": "git://github.com/csweichel/test-repo.git",
            "ssh_url": "git@github.com:csweichel/test-repo.git",
            "clone_url": "https://github.com/csweichel/test-repo.git",
            "svn_url": "https://github.com/csweichel/test-repo",
            "homepage": null,
            "size": 121,
            "stargazers_count": 0,
            "watchers_count": 0,
            "language": null,
            "has_issues": true,
            "has_projects": true,
            "has_downloads": true,
            "has_wiki": true,
            "has_pages": true,
            "forks_count": 1,
            "mirror_url": null,
            "archived": false,
            "disabled": false,
            "open_issues_count": 9,
            "license": null,
            "allow_forking": true,
            "is_template": false,
            "topics": [],
            "visibility": "public",
            "forks": 1,
            "open_issues": 9,
            "watchers": 0,
            "default_branch": "master",
            "stargazers": 0,
            "master_branch": "master"
        },
        "pusher": {
            "name": "csweichel",
            "email": "chris@gitpod.io"
        },
        "sender": {
            "login": "csweichel",
            "id": 3210701,
            "node_id": "MDQ6VXNlcjMyMTA3MDE=",
            "avatar_url": "https://avatars.githubusercontent.com/u/3210701?v=4",
            "gravatar_id": "",
            "url": "https://api.github.com/users/csweichel",
            "html_url": "https://github.com/csweichel",
            "followers_url": "https://api.github.com/users/csweichel/followers",
            "following_url": "https://api.github.com/users/csweichel/following{/other_user}",
            "gists_url": "https://api.github.com/users/csweichel/gists{/gist_id}",
            "starred_url": "https://api.github.com/users/csweichel/starred{/owner}{/repo}",
            "subscriptions_url": "https://api.github.com/users/csweichel/subscriptions",
            "organizations_url": "https://api.github.com/users/csweichel/orgs",
            "repos_url": "https://api.github.com/users/csweichel/repos",
            "events_url": "https://api.github.com/users/csweichel/events{/privacy}",
            "received_events_url": "https://api.github.com/users/csweichel/received_events",
            "type": "User",
            "site_admin": false
        },
        "installation": {
            "id": 5647067,
            "node_id": "MDIzOkludGVncmF0aW9uSW5zdGFsbGF0aW9uNTY0NzA2Nw=="
        },
        "created": false,
        "deleted": true,
        "forced": false,
        "base_ref": null,
        "compare": "https://github.com/csweichel/test-repo/compare/37cb8b8f7fc3...000000000000",
        "commits": [],
        "head_commit": null
    }
}