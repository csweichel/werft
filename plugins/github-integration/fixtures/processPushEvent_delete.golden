{"metadata":{"repository":{"host":"github.com","owner":"csweichel","repo":"test-repo","ref":"refs/heads/cw/tbd","defaultBranch":"master"},"trigger":"TRIGGER_DELETED"},"spec":{"repo":{"repo":{"host":"github.com","owner":"csweichel","repo":"test-repo","ref":"refs/heads/master","defaultBranch":"master"}},"repoSideload":[{"repo":{"host":"github.com","owner":"csweichel","repo":"test-repo","ref":"refs/heads/master","defaultBranch":"master"},"path":".werft"}]}}